### PR TITLE
changed port of discovery and removed comments from yml file

### PR DESCRIPTION
**Pull request by shilpa511raghav**
changes in yml file

### CODE REVIEW

File: Discovery/src/main/resources/application.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 server:
-  port: 9003
+  port: 8761
```
Comment (sourabhraghav2) [inline Discovery/src/main/resources/application.yml:2]: What is the reason to change the port

File: Gateway/src/main/resources/application.yml [hunk 2]
```diff
@@ -5,13 +5,6 @@ spring:
   application:
     name: Gateway
 
-#eureka:
-#  instance:
-#    hostname: localhost
-#  client:
-#    serviceUrl:
-#      defaultZone: http://localhost:9003/eureka/
-#
```
Comment (sourabhraghav2) [inline Gateway/src/main/resources/application.yml:14]: Please clean other unused/commented code also

Comment (sourabhraghav2) [reply Gateway/src/main/resources/application.yml:14]: Other

File: Discovery/src/main/resources/application.yml (hunk 1 above)
Comment (shilpa511raghav) [reply Discovery/src/main/resources/application.yml:2]: 8761 is default port for discovery so I changed it to the default port

**Pull request merged**